### PR TITLE
fixes #18; added configurable period for check execution;

### LINKED ISSUE #18: Checks should have a configurable execution period
Checks should have a configurable execution period to allow more flexible checks scheduling.

### PR DESCRIPTION
**Pull request by v-stepanov**
@antban @adyach @v1ctor @rcillo please review

### CODE REVIEW

File: bubuku/controller.py [hunk 1]
```diff
@@ -25,6 +25,20 @@ def can_run_at_exit(self) -> bool:
 
 
 class Check(object):
+
+    check_interval_s = 5
+    __last_check_timestamp_s = 0
+
+    def check_if_time(self) -> Change:
+        current_time_s = time()
+        if current_time_s - self.__last_check_timestamp_s >= self.check_interval_s:
```
Comment (rcillo) [inline bubuku/controller.py]: Maybe you could replace this by the function bellow `if time_till_check <= 0` ?

Comment (v-stepanov) [inline bubuku/controller.py]: 👍 good idea

File: bubuku/controller.py [hunk 2]
```diff
@@ -25,6 +25,19 @@ def can_run_at_exit(self) -> bool:
 
 
 class Check(object):
+
+    check_interval_s = 5
+    __last_check_timestamp_s = 0
```
Comment (antban) [inline bubuku/controller.py]: It's a class property, are you sure to add this property statically to all checks?

Comment (v-stepanov) [inline bubuku/controller.py]: fixed

**Comment by antban** [comment]
:+1:

**Pull request merged**